### PR TITLE
1022 replace vdsveh length 15min 1m bin aggregation with more useful MTO vehicle class aggregation

### LINKED ISSUE #1022: Replace `vds.veh_length_15min` 1m bin aggregation with more useful MTO vehicle length classification

### PR DESCRIPTION
**Pull request by gabrielwol**
## What this pull request accomplishes:

- Simple change to replaces `length_meter` in `vds.veh_length_15min` summary table with `mto_class_uid`
- add `traffic.mto_length_bin_classification`
- note: tested insert and select in gwolofs schema: `gwolofs.veh_length_15min`

## Issue(s) this solves:

- Closes #1022

## What, in particular, needs to reviewed:

- 

## What needs to be done by a sysadmin after this PR is merged
- truncate `vds.veh_length_15min`, make adjustments to schema and re-summarize (may take a few hours).

### CODE REVIEW

File: volumes/short_term_counting_program/sql/create-table-mto_length_bin_classification.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+--DROP TABLE IF EXISTS traffic.mto_length_bin_classification;
+
+CREATE TABLE traffic.mto_length_bin_classification
```
Comment (chmnata) [inline volumes/short_term_counting_program/sql/create-table-mto_length_bin_classification.sql:3]: where do the numbers come from? also can you fluff a liiitle + keep the decimal places consistent in the numrange

Comment (gabrielwol) [reply volumes/short_term_counting_program/sql/create-table-mto_length_bin_classification.sql:3]: This image... don't have a link to the actual source document but I'll stick it in the readme! 💡 
image

**Comment by radumas** [comment]
I'd seen elsewhere concerns about the quality of the length data and wanted to confirm that there was appropriate warning in the documentation/table comments

**Review by chmnata — APPROVED** [review]
:gabe-approved:

**Comment by gabrielwol** [comment]
> I'd seen elsewhere concerns about the quality of the length data and wanted to confirm that there was appropriate warning in the documentation/table comments

I will resolve this before merging.

**Comment by gabrielwol** [comment]
Successfully implemented all the changes ✅

**Pull request merged**